composer.json

```diff
@@ -26,7 +26,7 @@
     "require-dev": {
         "doctrine/collections": "^1.0",
         "doctrine/common": "^2.6",
-        "phpunit/phpunit": "^4.1"
+        "phpunit/phpunit": "^4.8.35"
```

Comment:
Could we upgrade straight to PHPUnit 6.0 rather?

Reply:
[`PHPUnit 6` requires `PHP 7`](https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-6.0.md#removed), but we can upgrade to version `5`, that [supports `PHP 5.6`](https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-5.0.md#removed-1).